server-user-stories-intro.md

```diff
@@ -0,0 +1,43 @@
+# User Stories for the Solid Server
+
+To enable more people to get involved, to improve code quality to
+production grade and to accelerate development, I would like to hear
```

Comment:
"grade and" should have a comma, as "grade, and"

Reply:
thanks!